### DISCUSSION TITLE
Any capture devices that capture Closed Caption (CC1) that work with SageTV or even NextPVR?

### DISCUSSION THREAD

**Discussion by graywolfusmc** [OpenSageTV/Community#96] [Capture Devices]
Any capture devices that capture Closed Caption (CC1) that work with SageTV or even NextPVR where the CC can be turned on/off and not burned into the video?

Since DirecTV is starting to remove SD channels again, looks like I'll need to replace my HVR-1600 capture cards with something new to capture HD since will eventually need to replace the SD DirecTV boxes.

**Comment by emveepee** [comment]
No capture devices could do this, closed captioning is only embedded in the ATSC and old NTSC signals you are still using.  

You might have some luck with ChannelsDVR  TV Everywhere channels for some channels that are not OTA.

For NextPVR I did create a way of toggling the OSD subtitles to my STB and Android device off and on but it is only useful for live TV and inevitably you will get recordings with them burnt in if you forget to turn them off.

**Comment by eelpout** [comment]
> Any capture devices that capture Closed Caption (CC1) that work with SageTV or even NextPVR where the CC can be turned on/off and not burned into the video?
> 

Silicon Dust [HDHomeRun Prime's](https://www.silicondust.com/product/hdhomerun-prime), which work with CableCARDs, do. I think Amazon still has some but I'm not clear if SD is still actively making them or just selling what they have left. 

the tricky part is getting a cable provider to give you a CableCARD these days. ;)

**Reply by emveepee** [reply]
A cable card device is also not a STB capture device from a DirectTV box that the user was looking for.

**Comment by eelpout** [comment]
> STB capture device from a DirectTV box

Hauppauge actually looked into this if I remember right when their HD-PVR with the analog video inputs first came out. the captioning is there in the VBI of the NTSC signal. they decided it wasn't worth putting in the engineering work to decode it in the HD-PVR (though they could have and had it in other products). I think today's "modern" HDMI DirecTV boxes have the CC in their own format, so it won't work anyway but I could be wrong.

in my defense I was answering more broadly to the _overall_ question in the title that yes, there were capture devices that did save the closed captioning info in the recorded file. :) I guess I should have made that clearer, that wouldn't work in their case in my response. 

I have both an HDHR Prime on cable and an HD-PVR for my DirecTV stuff so some of my recordings have captioning and some don't. :/

**Comment by graywolfusmc** [comment]
Thanks all for your comments and information. Guess the home PVR systems for HD just isn't right for the hard of hearing/deaf. Once DirecTV stops supporting SD altogether or at least the channels we care about then probably pull the plug and use Sage for the "library" my wife has built up and ripped DVDs. Fortunately our local Library has a large DVD collection.

**Comment by TGsell1** [comment]
My OTA reception is excellent and using the SD quad tuner.   Ccs work perfect through the HD300.  Not so through firestick miniclient since I’m using transcoding there and it strips out the CC feed.   Sent from my iPhoneOn Dec 1, 2023, at 11:30 AM, KeithAbbott ***@***.***> wrote:﻿
Depending on your OTA reception, an antenna with corresponding HDHomeRun equipment may be an option for broadcast channels.

Server Hardware: MSI Z270 SLI Plus ATX Motherboard, Intel i7-7700T CPU, 32GB Memory
Server Software: Unraid 6.12.4, sagetvopen-sagetv-server-opendct-java11 Docker (version 2.0.7)
Tuners: 2 x SiliconDust HDHomeRun Prime Cable TV Tuners, SiliconDust HDHomeRun CONNECT 4K OTA Tuner
Clients: Multiple HD300 Extenders, Multiple Fire TV Stick 4K Max w/MiniClient
Miscellaneous: Multiple Sony RM-VLZ620 Universal Remote Controls
